projects/plugins/jetpack/extensions/blocks/slideshow/style.scss

```
@@ -38,6 +38,22 @@
 			opacity: 1;
 		}
 
+		// Prevent layout shifts during loading, by using serialized aspect ratio.
+		// These rules mimic the conditions in swiperResize() on swiper-callbacks.js.
+		&:not(.wp-swiper-initialized) .wp-block-jetpack-slideshow_swiper-wrapper {
+      aspect-ratio: max(min(var(--aspect-ratio), calc(16 / 9)), 1);
+      max-height: 80vh;
+    }
+
+    // For older slideshows, where the aspect ratio wasn't serialized,
+    // we can partially mitigate layout shifts during loading.
+    // Since the aspect ratio is always calculated based on the first slide,
```

```suggestion
			aspect-ratio: max(min(var(--aspect-ratio), calc(16 / 9)), 1);
			max-height: 80vh;
		}

		// For older slideshows, where the aspect ratio wasn't serialized,
		// we can partially mitigate layout shifts during loading.
		// Since the aspect ratio is always calculated based on the first slide,
```
I'm not sure I've got the spacing quite right here, but I noticed it was off (using spaces instead of tabs).